pkg/cachemanager/cachemanager_test/cachemanager_integration_test.go

```diff
@@ -35,92 +35,137 @@ const (
 
 var cfg *rest.Config
 
+var (
+	configMapGVK = schema.GroupVersionKind{Group: "", Version: "v1", Kind: "ConfigMap"}
+	podGVK       = schema.GroupVersionKind{Group: "", Version: "v1", Kind: "Pod"}
+
+	configInstancOne      = "config-test-1"
+	configInstancTwo      = "config-test-2"
```

Comment:
Better to not use the word "Config" by itself to refer to a ConfigMap since the `Config` resource is also mixed up in this domain.

`cm1Name` or `configMap1Name` or something like that would be ideal.  Then you won't have variable contention if you need to make the objects:

```
configMap1 := fakes.ConfigMap(configMap1Name, ...)
// etc
```

Reply:
renamed.